### PR TITLE
New Bluej Commands Cheat Sheet

### PR DESCRIPTION
**Pull request by aviatorpranoy**
## People to notify

---

Instant Answer Page: https://duck.co/ia/view/bluej_commands_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@aviatorpranoy Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/blue-jcheats.json [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+        "sourceName": "",
+
+        "sourceUrl" : ""
+
+    },
+
+
+
+    "aliases": [
```
Comment (marti1125) [inline share/goodie/cheat_sheets/json/blue-jcheats.json]: Hi! error in this:
FAIL: Uppercase detected in alias - _aliases should be lowercase_

**Comment by marti1125** [comment]
Hi @aviatorpranoy this cheatsheet has a source ? Bluej documentations maybe?

**Comment by marti1125** [comment]
remove: Submodule zeroclickinfo-goodies added at 4f361a only push .json file

**Comment by marti1125** [comment]
Hi @aviatorpranoy any progress with this =D

File: share/goodie/cheat_sheets/json/blue-jcheats.json [hunk 2]
```diff
@@ -0,0 +1,223 @@
+{
+
+    "id": "bluej_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/blue-jcheats.json]: @aviatorpranoy this **must** match the ID of your IA page: `bluej_commands_cheat_sheet`

likewise the _filename_ must **also** follow suite: `bluej-commands.json`

Please change these so we can deploy your CheatSheet to beta 👍

**Comment by moollaza** [comment]
@aviatorpranoy @marti1125 I've corrected the ID for this Cheat Sheet.

The new IA Page url is https://duck.co/ia/view/bluej_commands_cheat_sheet -- i've update the PR description link for you 👍

**Comment by pjhampton** [comment]
Looks like this has gone stale. Closing for now

**Pull request closed without merging**